ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -2117,16 +2117,6 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-gcp-ipi-baselinecaps-none-additionalenabledcaps
-- as: gcp-ipi-cco-manual-workload-identity-p2-f7
```

Comment:
What is the  reason of removing this job?

Reply:
FYI "_...because gcp-ipi-disconnected-private-sts-p2-f7 has covered the scenario_", I only tell the reason in the first one listed, in each category. Thanks!

Reply:
FYI "_...because gcp-ipi-disconnected-private-sts-p2-f7 has covered the scenario_", I only tell the reason in the first one listed, in each category. Thanks!

Reply:
I thought `gcp-ipi-cco-manual-workload-identity-p3-f28-destructive` can be removed together, seem like `gcp-ipi-disconnected-private-sts` does not have destructive testing, so it is okay for me now.